src/components/DeploymentDetails.vue

```diff
@@ -6,9 +6,15 @@
       </template>
     </p-key-value>
 
-    <p-key-value label="Work Queue" :alternate="alternate">
-      <template v-if="deployment.workQueueName" #value>
-        <WorkQueueIconText :work-queue-name="deployment.workQueueName!" />
+    <p-key-value v-if="deployment.workPoolName" label="Work Pool" :alternate="alternate">
+      <template #value>
+        <WorkPoolIconText :work-pool-name="deployment.workPoolName!" />
+      </template>
+    </p-key-value>
+
+    <p-key-value v-if="deployment.workQueueName" label="Work Queue" :alternate="alternate">
```

Comment:
I'm not sure if you should add `&& deployment.workPoolName` to this `v-if`, or if it's okay to display the `WorkQueueIconText` without it

Reply:
It's good question. For now `work-pool-name` prop is optional, so technically, we don't need to have the pool name to at least show the name.  I wonder if even old deployments will have work pool name added to it right away